scan-payment/src/main/java/com/getbouncer/scan/payment/ml/SSDOcr.kt

```diff
@@ -88,11 +88,15 @@ private val FEATURE_MAP_SIZES =
  */
 private val PRIORS = combinePriors()
 
+data class PaymentCardOcrResult(val pan: String?, val name: String?, val expiry: String?)
```

Comment:
looks like a rebase gone wrong? this should not exist, it was replaced by `Prediction` below.

Reply:
this should actually be a part of cardscan-ui, moved it there